extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -256,9 +275,33 @@ oppia.directive('graphViz', [
           $scope.VERTEX_RADIUS = graphDetailService.VERTEX_RADIUS;
           $scope.EDGE_WIDTH = graphDetailService.EDGE_WIDTH;
 
+          if ($scope.canEditGraph === undefined ||
```

Comment:
I think it's OK to pass $scope.interactionIsActive directly. I don't think you need canEditGraph, it just adds indirection.

Reply:
Done, changed that.